kolena/io.py

```diff
@@ -135,6 +135,34 @@ def dataframe_from_json(*args: Any, **kwargs: Any) -> pd.DataFrame:
     return df_post
 
 
+def dataframe_from_jsonl(*args: Any, **kwargs: Any) -> pd.DataFrame:
+    """
+    Load pandas DataFrame from JSON Lines file or string.
+    Handles errors in JSON parsing to provide clearer feedback on failures.
+
+    :param args: positional arguments for `pandas.DataFrame.read_json`.
+    :param kwargs: keyword arguments for `pandas.DataFrame.read_json`.
+    :return: DataFrame.
+    """
+    df = pd.read_json(*args, **kwargs, lines=True)
+    df_post = _dataframe_object_serde(df, _deserialize_dataobject)
+    return df_post
+
+
+def dataframe_to_jsonl(df: pd.DataFrame, *args: Any, **kwargs: Any) -> str:
```

Comment:
Do we need this? Users should be knowledgeable to just call `df.to_json(lines=True)`.

Reply:
I'm in favor of not addition new helpers unless there's something existing ones cannot do. Thoughts?

Reply:
Ok to remove, if users find necessary we can bring in the future.